x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/integrations.ts

```diff
@@ -47,6 +47,7 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
   describe('Integrations', () => {
     let adminRoleAuthc: RoleCredentials;
     let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
+    let preExistingIntegrations: List<string>;
```

Comment:
```suggestion
    let preExistingIntegrations: string[];
```

`List` is not a standard TypeScript type, in particular case we can go simple

Reply:
From CI: 

```

2024-11-07 16:04:37 UTC | proc [tsc] x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/integrations.ts:50:34 - error TS2304: Cannot find name 'List'.
-- | --
  | 2024-11-07 16:04:37 UTC | proc [tsc]
  | 2024-11-07 16:04:37 UTC | proc [tsc] 50     let preExistingIntegrations: List<string>;
```

You probably need to configure ESLint locally to catch it before opening PR.